latch_cli/centromere/utils.py

```diff
@@ -168,9 +166,9 @@ def _from_env():
 
 
 def _construct_ssh_client(host_ip: str, username: str):
-
     ssh = paramiko.SSHClient()
     ssh.load_system_host_keys()
+    ssh.set_missing_host_key_policy(paramiko.MissingHostKeyPolicy())
```

Comment:
```suggestion
    ssh.set_missing_host_key_policy(paramiko.WarningPolicy())
```

`paramiko.MissingHostKeyPolicy()` is an interface and u should use a subclass  of it - i think `paramiko.WarningPolicy` (accept but log a warning) over `paramiko.AutoAddPolicy` (accept without any warnings) is appropriate here